admin/starter/.github/workflows/close-pull-request.yml

```diff
@@ -2,16 +2,23 @@ name: Close Pull Request
 
 on:
   pull_request_target:
-    types: [opened]
+    types: [opened, reopened]
+
+permissions:
+  pull-requests: write
 
 jobs:
   main:
     runs-on: ubuntu-latest
     steps:
       - name: Close PR with nice message
-        uses: superbrothers/close-pull-request@v3
-        with:
-          comment: >
+        run: |
+          cd ${{ github.workspace }}
+          gh pr close $ISSUE -c "$COMMENT"
```

Comment:
While definitely this is run on bash, I would rather like to see ISSUE and COMMENT step env accessed as env variables on Github. like ${{ env.ISSUE }}